.vib/prestashop/cypress/cypress.json

```diff
@@ -0,0 +1,14 @@
+{
+  "env": {
+    "username": "user",
+    "email": "user@example.com",
```

Comment:
As we are using this param actively in the tests, we should define it explicitly in the chart values just like the user and password.